app/uk/gov/hmrc/statepension/controllers/auth/AuthAction.scala

```diff
@@ -44,7 +44,9 @@ class AuthActionImpl @Inject()(val authConnector: AuthConnector)(implicit execut
       Future.successful(Some(BadRequest))
     } else {
       val uriNino: Option[String] = Some(matches.group(1))
-      authorised((ConfidenceLevel.L200 and Nino(true, uriNino)) or AuthProviders(PrivilegedApplication)) {
+      authorised(
+        (ConfidenceLevel.L200 and Nino(true, uriNino)) or (ConfidenceLevel.L500 and AuthProviders(Verify)) or AuthProviders(PrivilegedApplication)
```

Comment:
So this says as long as you login in with verify you can't access anyone nino...? Is that correct

Reply:
if you come in through verify you can use any Nino otherwise you can only use your own